Sources/Clappr_iOS/Classes/Base/Playback.swift

```diff
@@ -120,3 +120,26 @@ open class Playback: UIBaseObject, Plugin {
         Logger.logDebug("destroyed", scope: "Playback")
     }
 }
+
+// MARK: - DVR
+extension Playback {
+    @objc var minDvrSize: Double {
+        return 0
+    }
+
+    @objc var usingDVR: Bool {
+        return false
+    }
+
+    @objc open var seekableTimeRanges: [NSValue] {
```

Comment:
Why do we need to expose the seekableTimeRange in this way? Could we be more specific? Like a ClosedRange for example.